### PR TITLE
fixes misc pdf render issues

### PR DESCRIPTION
**Pull request by esotrope**
PDF Edge cases tested.

screen shot 2017-07-29 at 10 31 09 pm
screen shot 2017-07-29 at 10 34 50 pm
screen shot 2017-07-29 at 10 32 08 pm
screen shot 2017-07-29 at 10 32 51 pm

### CODE REVIEW

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/report/ReportViewSupport.java [hunk 1]
```diff
@@ -76,21 +76,24 @@ public String round(final float value) {
...
+                minimumScore,
                 // The second and third cut point are the middle cut point +/- the product of the individual exam
-                // result scale score standard error * 1.5
-                toIntExact(assessment.getCutPoints().get(2) - Math.round(exam.getScaleScore().getStandardError() * 1.5)),
-                toIntExact(assessment.getCutPoints().get(2) + Math.round(exam.getScaleScore().getStandardError() * 1.5)),
-                assessment.getCutPoints().get(4)
+                // result scale score standard error and 1.5
+                // this value must be clamped to be within the bounds of the minimum and maximum score
+                clamp(toIntExact(assessment.getCutPoints().get(2) - Math.round(exam.getScaleScore().getStandardError() * 1.5)), minimumScore, maximumScore),
+                clamp(toIntExact(assessment.getCutPoints().get(2) + Math.round(exam.getScaleScore().getStandardError() * 1.5)), minimumScore, maximumScore),
```
Comment (esotrope) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/report/ReportViewSupport.java]: @jtreuting I was only able to reproduce the bars showing on multiple rows when the width added to greater than 100% - this clamp should ensure that can't happen, though I would like to retest with the user you used in qa/uat.

File: common/src/main/resources/templates/fragments.html [hunk 2]
```diff
@@ -5,18 +5,21 @@
...
...G9yOiM2RkFFMjk7Ym9yZGVyLWNvbG9yOiM2Mjk5MjR9LmJ0bi1zdWNjZXNzLmZvY3VzLC5idG4tc3VjY2Vzczpmb2N1c3tjb2xvcjojZmZmO2JhY2tncm91bmQtY29sb3I6IzU1ODUxZjtib3JkZXItY29sb3I6IzIwMzIwY30uYnRuLXN1Y2Nlc3MuYWN0aXZlLC5idG4tc3VjY2VzczphY3RpdmUsLmJ0bi1zdWNjZXNzOmhvdmVyLC5vcGVuPi5kcm9wZG93bi10b2dnbGUuYnRuLXN1Y2Nlc3N7Y29sb3I6I2ZmZjtiYWNrZ3JvdW5kLWNvbG9yOiM1NTg1MWY7Ym9yZGVyLWNvbG9yOiM0MjY4MTh9LmJ0bi1zdWNjZXNzLmFjdGl2ZS5mb2N1cywuYnRuLXN1Y2Nlc3MuYWN0aXZlOmZvY3VzLC5idG4tc3VjY2Vzcy5hY3RpdmU6aG92ZXIsLmJ0bi1zdWNjZXNzOmFjdGl2ZS5mb2N1cywuYnRuLXN1Y2Nlc3M6YWN0aXZlOmZvY3VzLC5idG4tc3VjY2VzczphY3RpdmU6aG92ZXIsLm9wZW4+LmRyb3Bkb3duLXRvZ2dsZS5idG4tc3VjY2Vzcy5mb2N1cywub3Blbj4uZHJvcGRvd24tdG9nZ2xlLmJ0bi1zdWNjZXNzOmZvY3VzLC5vcGVuPi5kcm9wZG93bi10b2dnbGUuYnRuLXN1Y2Nlc3M6aG92ZXJ7Y29sb3I6I2ZmZjtiYWNrZ3JvdW5kLWNvbG9yOiM0MjY4MTg7Ym9yZGVyLWNvbG9yOiMyMDMyMGN9LmJ0bi1zdWNjZXNzLmFjdGl2ZSwuYnRuLXN1Y2Nlc3M6YWN0aXZlLC5vcGVuPi5kcm9wZG93bi10b2dnbGUuYnRuLXN1Y2Nlc3N7YmFja2dyb3VuZC1pbWFnZTpub25lfS5idG4tc3VjY2Vzcy5kaXNhYmxlZC5mb2N1cywuYnRuLXN1Y2Nlc3MuZGlzYWJsZWQ6Zm9jdXMsLmJ0bi1zdWNjZXNzLmRpc2FibGVkOmhvdmVyLC5idG4tc3VjY2Vzc1tkaXNhYmxlZF0uZm9jdXMsLmJ0bi1zdWNjZXNzW2Rpc2FibGVkXTpmb2N1cywuYnRuLXN1Y2Nlc3NbZGlzYWJsZWRdOmhvdmVyLGZpZWxkc2V0W2Rpc2FibGVkX
```
Comment (esotrope) [inline common/src/main/resources/templates/fragments.html]: Directly embedded latest version of sbac-ui-kit-wkhtmltopdf.min.css with latest additions.
I'm hoping the docker image can pull this down and keep it locally eventually.

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/report/ReportViewSupport.java (hunk 1 above)
Comment (jtreuting) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/report/ReportViewSupport.java]: Yeah that makes sense, for the average scale score display I had to adjust the totals after rounding to make sure they summed to 100.  I can point out that code if it would be helpful.  

I'll send the user for UAT in slack to you.

Comment (esotrope) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/report/ReportViewSupport.java]: cool I'd like to compare impls thanks

**Comment by esotrope** [comment]
Test result for crazy large stderr edge case:
screen shot 2017-07-29 at 11 07 44 pm
